infrastructure/src/main/resources/templates/dozenten/ersteller.html

```diff
@@ -26,20 +26,14 @@ <h1 class="display-4 text-center">Dozenten-Übersicht</h1>
             <div class="card-body">
                 <form class="form-horizontal was-validated" method="post" th:action="@{/feedback/dozenten/new}">
                     <div class="form-group row">
-                        <label for="veranstaltung"></label>
+                        <label for="veranstaltung">Veranstaltung:</label>
                         <br>
                         <div class="input-group mb-3 col-sm-6">
-                            <input class="form-control" id="veranstaltung" name="veranstaltung"
-                                   placeholder="Name der Veranstaltung" required type="text">
-                        </div>
-                    </div>
-                    <hr class="m-5">
-                    <div class="form-group row">
-                        <label for="dozentname"></label>
-                        <br>
-                        <div class="input-group mb-3 col-sm-6">
-                            <input class="form-control" id="dozentname" name="dozentname"
-                                   placeholder="Name des Dozenten" required type="text">
+                            <select id="veranstaltung" name="veranstaltung" required>
+                                <div class="foo" th:each="veranstaltung : ${veranstaltungen}">
+                                    <option th:value="${veranstaltung.veranstaltungsNr}" th:text="${veranstaltung.name}">Vorlesung</option>
+                                </div>
+                            </select>
```

Comment:
Not sure why you removed so much. This looks a bit broken when I run the application so I am not sure if this is intentional.

Reply:
The lecturer's name is inserted automatically to the questionnaire. So the input for the name is unnecessary. The event is selected by the dropdown menu (lines 32-36).

I really think, it looks broken when you run the application, because the UUID of "orga1" is different from the UUID that is entered in all the events.  If the UUIDs are equal, all events of the lecturer are entered in the dropdown menu.

Reply:
Alright fair enough